Nybble/nybble_modules/create_alert.py

```diff
@@ -20,19 +20,36 @@ class NybbleAlert(dict):
     Final Object sent to Nybble Hub
     """
 
-    alert_processing_time: str
-    alert_fields: dict[str, str]
-    alert_event_original: dict
-    alert_id: str
-    alert_level: int
-    rule_status: str
-    rule_name: str
-    rule_tags: list[str]
-    rule_references: list[str]
-    sigma_rule_id: str
-    mssp_client: str
-    custom_fields_visible: dict
-    custom_fields_hidden: dict
+    def __init__(
+        self,
+        alert_processing_time=None,
+        alert_fields=None,
+        alert_event_original=None,
+        alert_id=None,
+        alert_level=None,
+        rule_status=None,
+        rule_name=None,
+        rule_tags=None,
+        rule_references=None,
+        sigma_rule_id=None,
+        mssp_client=None,
+        custom_fields_visible=None,
+        custom_fields_hidden=None,
+    ):
+        super().__init__()
+        self["alert_processing_time"] = alert_processing_time
+        self["alert_fields"] = alert_fields if alert_fields is not None else {}
+        self["alert_event_original"] = alert_event_original if alert_event_original is not None else {}
+        self["alert_id"] = alert_id
+        self["alert_level"] = alert_level
+        self["rule_status"] = rule_status
+        self["rule_name"] = rule_name
+        self["rule_tags"] = rule_tags if rule_tags is not None else []
+        self["rule_references"] = rule_references if rule_references is not None else []
+        self["sigma_rule_id"] = sigma_rule_id
+        self["mssp_client"] = mssp_client
+        self["custom_fields_visible"] = custom_fields_visible if custom_fields_visible is not None else {}
+        self["custom_fields_hidden"] = custom_fields_hidden if custom_fields_hidden is not None else {}
```

Comment:
A [pydantic model](https://docs.pydantic.dev/latest/concepts/models/) can be used instead (with default values for the fields), with a [serialization](https://docs.pydantic.dev/latest/concepts/serialization/) in the request call